### ISSUE TITLE
Internal error occured , can't use /dxl leave

### ISSUE THREAD

**Issue by MorenaDaniel** [DRE2N/DungeonsXL#175]
EDIT : **I saw you answer to my last post but adding DXL to multiverse doesn't change anything, and same when i remove them :/**

Hi ! I''ve reinstalled DungeonsXl and when i try to make my first dungeon i use /dxl create test and when i use /dxl leave i have a internal error occured. I can't leave the dungeon, except via /mvtp command. I looks like my Health is generating the problem. I'm using Item Lore Stats and MythicMobs that give me extra health. How can i change the max health of DungeonsXL to 200000 ? 

```
Erisium_Nope issued server command: /dxl leave
[04:07:22 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'dxl' in plugin DungeonsXL v0.15
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at org.bukkit.craftbukkit.v1_10_R1.CraftServer.dispatchCommand(CraftServer.java:646) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.PlayerConnection.handleCommand(PlayerConnection.java:1351) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:1186) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_111]
        at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_111]
        at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:733) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:399) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:672) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:571) [spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_111]
Caused by: java.lang.IllegalArgumentException: Health must be between 0 and 20.0
        at org.bukkit.craftbukkit.v1_10_R1.entity.CraftLivingEntity.setHealth(CraftLivingEntity.java:95) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        at io.github.dre2n.dungeonsxl.player.DGlobalPlayer.reset(DGlobalPlayer.java:285) ~[?:?]
        at io.github.dre2n.dungeonsxl.player.DEditPlayer.leave(DEditPlayer.java:145) ~[?:?]
        at io.github.dre2n.dungeonsxl.command.LeaveCommand.onExecute(LeaveCommand.java:84) ~[?:?]
        at io.github.dre2n.dungeonsxl.util.commons.command.BRCommandExecutor.onCommand(BRCommandExecutor.java:71) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.10.2-R0.1-SNAPSHOT-latest.jar:git-Spigot-5391d73-0ebb9c7]
        ... 15 more
```

**Comment by Sataniel98** [comment]
I'll fix it in the next patch.

**Comment by Sataniel98** [comment]
Should be fixed in 0.15.1 build 3, but please give me some feedback :)
